src/main/java/org/embulk/output/GcsOutputPlugin.java

```diff
@@ -78,6 +88,26 @@ public TransactionalFileOutput open(TaskSource taskSource, final int taskIndex)
         return new GcsTransactionalFileOutput(task, client, taskIndex);
     }
 
+    private void deleteFiles(PluginTask task)
+    {
+        logger.info("delete files");
+        Storage client = createClient(task);
+        try {
+            Objects listResult = client.objects().list(task.getBucket()).setDelimiter("/").setPrefix(task.getPathPrefix()).execute();
+            if (listResult.getItems() == null) {
+                logger.info("no files found");
+                return;
+            }
+            for (StorageObject item : listResult.getItems()) {
+                logger.info("delete file: {}/{}", item.getBucket(), item.getName());
+                client.objects().delete(item.getBucket(), item.getName()).execute();
+            }
+        }
+        catch (ConfigException | IOException ex) {
+            throw Throwables.propagate(ex);
```

Comment:
I don't think ConfigException is thrown from any part of this method.
But it would be nice to throw ConfigException if IOException happens.